ci/check-documentation.py

```diff
@@ -86,6 +88,22 @@ def complain(message):
         # check local link validity
 
         for url in sorted([ref.url for ref in doc_page.external_references()]):
+            if url.startswith(omz_github_url):
+                suggested_reference = url[len(omz_github_url):]
+
+                if suggested_reference.startswith(('blob', 'tree')):
+                    md_path_parent_dir_components = '../' * len(md_path_rel.parent.parts)
```

Comment:
This algorithm is rather crude, the suggested URL that it builds goes all the way up to the root directory and then down to the target file. That's usually not a good suggestion. For example, if `a/b/c.md` links to `https://github.com/openvinotoolkit/open_model_zoo/blob/a/b/d.md`, then it should be replaced with just `d.md`, not `../../a/b/d.md`.